src/NHibernate/Cache/StandardQueryCache.cs

```diff
@@ -31,20 +31,13 @@ public partial class StandardQueryCache : IQueryCache, IBatchableQueryCache
 			string regionName)
 			: this(
 				updateTimestampsCache,
-				settings.CacheProvider.BuildCache(
-					(!string.IsNullOrEmpty(settings.CacheRegionPrefix) ? settings.CacheRegionPrefix + '.' : "") +
-					(regionName ?? typeof (StandardQueryCache).FullName),
+				CacheFactory.BuildCacheBase(
+					settings.GetFullCacheRegionName(regionName ?? typeof(StandardQueryCache).FullName),
+					settings,
 					props))
 		{
 		}
 
-		// Since v5.2
-		[Obsolete]
-		private StandardQueryCache(UpdateTimestampsCache updateTimestampsCache, ICache cache)
```

Comment:
Introduced in 5.2, so just remove it.

Reply:
Moreover, private. But I think than previously, this constructor was used by an obsoleted exposed member. Obviously it is no more the case.